presto-hive/src/main/java/io/prestosql/plugin/hive/S3SelectPushdown.java

```diff
@@ -50,9 +49,10 @@
  * S3SelectPushdown uses Amazon S3 Select to push down queries to Amazon S3. This allows Presto to retrieve only a
  * subset of data rather than retrieving the full S3 object thus improving Presto query performance.
  */
-public class S3SelectPushdown
+public final class S3SelectPushdown
 {
-    private static final Logger LOG = Logger.get(S3SelectPushdown.class);
+    private S3SelectPushdown() {}
```

Comment:
Normally the constructor is placed after the constants.

Reply:
Indeed.... although it's weird to me. This constructor is just utility class marker, so having is somewhere in the middle is not useful.

Reply:
Understood, but normally we prefer consistency in cases like this.